components/card.css

```diff
@@ -19,11 +19,17 @@
 
 .card__bookmark {
   position: relative;
-  top: -6rem;
+  top: -6.3rem;
   left: 6.5rem;
   height: 50px;
 }
 
+.card__bookmark--clicked {
+  fill: transparent;
```

Comment:
transparency looks weird here, you may want to use white instead of transparency.